packages/ui/src/council/queries/council.graphql

```diff
@@ -217,11 +222,11 @@ query GetPastCouncil($id: ID!, $fromBlock: Int!, $toBlock: Int!) {
   budgetSpendingEvents(where: { inBlock_gte: $fromBlock, inBlock_lte: $toBlock }) {
     ...CouncilSpendingEventFields
   }
-  fundingRequestsApproved: proposalExecutedEvents(
+  fundingRequestsApproved: proposals(
     where: {
-      inBlock_gt: $fromBlock
-      inBlock_lt: $toBlock
-      proposal: { details_json: { isTypeOf_eq: "FundingRequestProposalDetails" } }
+      statusSetAtBlock_gt: $fromBlock
+      statusSetAtBlock_lt: $toBlock
+      details_json: { isTypeOf_eq: "FundingRequestProposalDetails" }
```

Comment:
The query should only return actually executed funding requests, and maybe just those approved between `$fromBlock` and `$toBlock` regardless from when they executed.